app/renderer/components/preferences/payment/bitcoinDashboard.js

```diff
@@ -341,7 +341,7 @@ class BitcoinDashboard extends ImmutableComponent {
                         {/* TODO: refactor button.js */}
                         <Button className={cx({
                           primaryButton: true,
-                          [css(globalStyles.spacing.dialogInsideMargin)]: true
+                          [css(styles.bitcoinAddressButton)]: true
```

Comment:
In BEM it should be `dialogInsideMargin--top` and `dialogInsideMargin--bottom` anyway.